src/averageAges.js

```diff
@@ -14,14 +14,17 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm'
+    && (!century || Math.ceil(person.died / 100) === century));
+
+  return averageAge(men);
 }
 
+const averageAge = peopleFilter => peopleFilter.reduce((sumAge, person) =>
+  sumAge + person.died - person.born, 0) / peopleFilter.length;
```

Comment:
It's a better wraping
```suggestion
const averageAge = (peopleFilter) => (
  peopleFilter.reduce((sumAge, person) => (
    sumAge + person.died - person.born, 0) / peopleFilter.length
  ));
```

Reply:
Change it in all cases